### PR TITLE
Use 7zip instead of LZMA to compress catalog

### PR DESCRIPTION
**Pull request by stephanheilner**

### CODE REVIEW

**Comment by nickmshelley** [comment]
After you get tests passing, can you test this as a development pod in GL to make sure it works before we merge?

**Comment by stephanheilner** [comment]
Been doing that (with development pod). This works. We actually have a test that downloads a catalog and extracts it, so if they pass, its working correctly.

**Comment by stephanheilner** [comment]
👎 Wait for a little more testing before merging...

File: LDSContent/Archive.swift [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+//
+
+import Foundation
+import LzmaSDK_ObjC
+
+public struct Archive {
+
+    public static func decompress7z(fromURL: URL, toDirectory: URL) throws {
+        let reader = LzmaSDKObjCReader(fileURL: fromURL, andType: LzmaSDKObjCFileType7z)
+        try reader.open()
```
Comment (nickmshelley) [inline LDSContent/Archive.swift]: Do you need to close this at the end?

File: LDSContent/Archive.swift [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        writer.compressionLevel = 9
+        writer.encodeContent = true
+        writer.encodeHeader = true
+        writer.compressHeader = true
+        writer.compressHeaderFull = true
+        writer.writeModificationTime = false
+        writer.writeCreationTime = false
+        writer.writeAccessTime = false
+        
+        try writer.open()
```
Comment (nickmshelley) [inline LDSContent/Archive.swift]: Do you need to close this after the write?

**Comment by nickmshelley** [comment]
.

**Comment by stephanheilner** [comment]
Using .lzma, opening new PR

**Pull request closed without merging**